newm-server/src/main/kotlin/io/newm/server/ktx/StringExt.kt

```diff
@@ -20,3 +22,11 @@ fun String?.asValidUrl(): String {
     if (!isValidUrl()) throw HttpUnprocessableEntityException("Invalid url: $this")
     return this
 }
+
+fun String.toReferenceUtxo(): Utxo {
+    val parts = this.split('#')
+    return utxo {
+        hash = parts[0]
+        ix = parts[1].toLong()
```

Comment:
Any possibility here of missing `#` and the split returning only one part?

Reply:
Not really possible. This value comes from the config table in our db. If we enter it wrongly, that's on us.